src/expr.cpp

```diff
@@ -1737,17 +1737,37 @@ bool lCreateBinaryOperatorCall(const BinaryExpr::Op bop, Expr *a0, Expr *a1, Exp
         std::string opName = std::string("operator") + lOpString(bop);
         std::vector<Symbol *> funs;
         m->symbolTable->LookupFunction(opName.c_str(), &funs);
-        if (funs.size() == 0) {
+        if (funs.size() > 0) {
+            Expr *func = new FunctionSymbolExpr(opName.c_str(), funs, sp);
+            ExprList *args = new ExprList(sp);
+            args->exprs.push_back(arg0);
+            args->exprs.push_back(arg1);
+            op = new FunctionCallExpr(func, args, sp);
+            return abort;
+        }
+
+        // templates
+        std::vector<TemplateSymbol *> funcTempls;
+        m->symbolTable->LookupFunctionTemplate(opName.c_str(), &funcTempls);
+        if (funcTempls.size() > 0) {
+            std::vector<std::pair<const Type *, SourcePos>> *vec = new std::vector<std::pair<const Type *, SourcePos>>;
```

Comment:
Seems that it needs to be a vector, not a pointer to a vector. This way no need to dereference when passing it and no need to delete.